docs/en/operation/Metrics-System.md

```diff
@@ -96,6 +97,43 @@ The filename will correspond with the metric name.
 
 Refer to `metrics.properties.template` for all possible sink specific configurations. 
 
+### Prometheus Sink Setup
+
+[Prometheus](https://prometheus.io/) is a monitoring tool that can help monitoring Alluxio metrics changes.
+
+In the metrics property file, `$ALLUXIO_HOME/conf/metrics.properties` by default, add the following properties:
```

Comment:
does the server need to be restarted after setting this property for it to work?

Reply:
Added the distributed and server restarted part.